Automattic/jetpack
# Add/jetpack starter cards

Fixes 1161179020265237-as-1204496174692410

## Proposed changes:

* This PR adds the plan header and plan body for Jetpack Starter to `/wp-admin/admin.php?page=jetpack#/my-plan`, and the post-activation screen.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






*  Check out this branch locally, build Jetpack, My Jetpack and Licensing. Or spin up an Atomic ephemeral site in pool 1 with Jetpack Beta installed, and choose this branch
* Follow the instructions here D109141-code
* Point your ephemeral site to your sandbox
* Add the license you have just purchased to your site
* Confirm that the activation screen looks like below
* Confirm that My Jetpack looks like the screenshot below

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/5714212/235252076-113cc166-ea7c-4178-852f-8ad84509c32e.png">

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/5714212/235266124-c6b003ee-c346-4968-b584-1961ba7ca51a.png">
